projects/demo/src/pages/kit/time/time-mask-doc.template.html

```diff
@@ -55,6 +55,32 @@
             </ng-template>
             <time-mask-doc-example-2 />
         </tui-doc-example>
+
+        <tui-doc-example
+            id="step"
+            heading="Arrows stepping"
+            [content]="stepExample3"
+            [description]="stepDescription"
+        >
+            <ng-template #stepDescription>
+                <p class="tui-space_top-0 tui-space_bottom-1">
+                    Property
+                    <code>step</code>
+                    allows you to increment and decrement time segments by pressing
+                    <code>UpArrow</code>
+                    and
+                    <code>DownArrow</code>
```

Comment:
```suggestion
                    <code>ArrowUp</code>
                    and
                    <code>ArrowDown</code>
```

(the same naming as `event.key` of `keydown` event https://w3c.github.io/uievents/#events-keyboard-key-location)